src/ProcessCommunicationTrait.php

```diff
@@ -93,7 +93,7 @@ protected function getSockFile($affix, $overwrite)
             $run .= '/';
         }
 
-        if (!is_dir($run) && !mkdir($run, 0777, true)) {
+        if (!is_dir($run) && !mkdir($run, 0777, true) && !is_dir($run)) {
```

Comment:
Why is this needed?

Reply:
Race conditions: https://github.com/kalessil/phpinspectionsea/blob/master/docs/probable-bugs.md#mkdir-race-condition

Reply:
Understood. Then that links needs to go into the source as the change makes the goal of that line hard to grasp?